doc/classes/OS.xml

```diff
@@ -297,30 +334,30 @@
 		<method name="get_name" qualifiers="const">
 			<return type="String" />
 			<description>
-				Returns the name of the host OS.
-				On Windows, this is [code]"Windows"[/code].
-				On macOS, this is [code]"macOS"[/code].
-				On Linux-based operating systems, this is [code]"Linux"[/code].
-				On BSD-based operating systems, this is [code]"FreeBSD"[/code], [code]"NetBSD"[/code], [code]"OpenBSD"[/code], or [code]"BSD"[/code] as a fallback.
-				On Android, this is [code]"Android"[/code].
-				On iOS, this is [code]"iOS"[/code].
-				On the web, this is [code]"Web"[/code].
-				[b]Note:[/b] Custom builds of the engine may support additional platforms, such as consoles, yielding other return values.
+				Returns the name of the host platform.
+				- On Windows, this is [code]"Windows"[/code].
+				- On macOS, this is [code]"macOS"[/code].
+				- On Linux-based operating systems, this is [code]"Linux"[/code].
+				- On BSD-based operating systems, this is [code]"FreeBSD"[/code], [code]"NetBSD"[/code], [code]"OpenBSD"[/code], or [code]"BSD"[/code] as a fallback.
+				- On Android, this is [code]"Android"[/code].
+				- On iOS, this is [code]"iOS"[/code].
+				- On the web, this is [code]"Web"[/code].
+				[b]Note:[/b] Custom builds of the engine may support additional platforms, such as consoles, possibly returning other names.
 				[codeblocks]
 				[gdscript]
 				match OS.get_name():
-				    "Windows":
-				        print("Windows")
+				    "Windows", "UWP":
+				        print("Welcome to Windows!")
 				    "macOS":
-				        print("macOS")
+				        print("Welcome to macOS!")
 				    "Linux", "FreeBSD", "NetBSD", "OpenBSD", "BSD":
-				        print("Linux/BSD")
+				        print("Welcome to Linux/BSD!")
 				    "Android":
-				        print("Android")
+				        print("This is an Android!")
 				    "iOS":
-				        print("iOS")
+				        print("This is an iOS!")
```

Comment:
Any specific reason why Android and iOS get a different pattern than the others?

Reply:
No, not particularly, in hindsight. Perhaps "Welcome to iOS" sounded off at first.